network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/current_events_slider_block.html

```diff
@@ -17,7 +17,7 @@ <h2 class="tw-h3-heading tw-font-semibold tw-mb-0 tw-w-full">{{ value.title }}</
         <div class="swiper-wrapper">
           {% for current_event in self.current_events %}
             {% image current_event.value.image fill-445x185 as img %}
-            <div class="swiper-slide">
+            <div class="swiper-slide tw-self-stretch">
```

Comment:
Can you add some context why we set stretch on every single item instead of setting this as the default for all children on the parent (`tw-align-stretch`)?

Reply:
@tbrlpld good call on updating styling on parent div instead of individual child divs. I was playing around with the child div code and didn't think much to touch other parts of the file (ie., parent div). I've updated the PR to reflect the new change (using `tw-items-stretch`).